changelog/7022.improvement.md

```diff
@@ -0,0 +1 @@
+Endpoints that require the existance of the conversation for the specified conversation id now raises an exception.
```

Comment:
Can we please list the changed endpoints in here? Ideally also which status code they now return.